ui/package.json

```diff
@@ -135,7 +135,7 @@
     "@influxdata/clockface": "2.3.1",
     "@influxdata/flux": "^0.5.1",
     "@influxdata/flux-lsp-browser": "^0.5.11",
-    "@influxdata/giraffe": "0.23.0",
+    "@influxdata/giraffe": "^0.24.0",
```

Comment:
No caret. Let's keep it to a specific version so that it doesn't get updated on accident by other PRs that yarn install.

Reply:
shoot, i think me saying 'do a `yarn add @influxdata/giraffe`, rather than modifying package.json' the other day is what caused this. sorry!

Reply:
i think this is my fault - i told rose to install via `yarn add @influxdata/giraffe`